src/cljs/org/broadinstitute/firecloud_ui/page/workspace/data/tab.cljs

```diff
@@ -57,12 +58,53 @@
                 [:div {:style style :onClick #(add-crumb :workspace-import "Choose Workspace")}
                  "Copy from another workspace"]]))]])}])})
 
+(react/defc EntityAttributes
+  {:render (fn [{:keys [props state]}]
+      (let [workspace (:workspace props)
+            bucket-name (get-in workspace [:workspace :bucketName])
+            attributes (:attributes props)
+            entity-name (:selected-entity props)
+            entity-type (str (:selected-entity-type props))]
+      (when (not= entity-type ":sample_set")
```

Comment:
There are no attributes for the sample sets, so we don't show attributes at all when looking at them